client.go

```diff
@@ -110,6 +113,67 @@ func (c *Client) SetUserAgent(ua string) *Client {
 	return c
 }
 
+type RequestParams struct {
+	Body     map[string]any
```

Comment:
Go's JSON encoder uses reflection to figure out the underlying type of an `any` object, so we can change `Body` to `any` to support directly serializing structs 👍

Reply:
Thanks for the tip! I fixed this 👍